instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests/ui/routeline/RouteLineLayersTest.kt

```diff
@@ -166,6 +168,7 @@ class RouteLineLayersTest : BaseTest<BasicNavigationViewActivity>(
     // for the primary route should be moved to the top of the stack above the other
     // route line layers.
     @Test
+    @UiThreadTest
```

Comment:
We cannot run the whole test on the main thread. The test includes a latch so as soon as we reach `countDownLatch.await()` I would imagine that the main thread deadlocks and the test should would just time out.

Instead, we should move the contents that need to be run on the main thread (interactions with the SDK) into a `runOnMainSync` block, like:
```
@Test
fun test() {
  val latch: Latch
  runOnMainSync {
    // interact with SDK and count down
  }
  latch.await()
}
```

There are many examples of that across other tests.

Reply:
I forgot about this.